pkg/arvo/app/link-listen-hook.hoon

```diff
@@ -399,8 +399,11 @@
 ++  leave-from-peer
   |=  [=app-path =group-path who=ship]
   ^-  (quip card _state)
+  ::  we left/got kicked from the group
+  ::  so leave collection
```

Comment:
Because this is logic specific to group changes, and not part of "ignoring a peer" per se, I think this is better at home in `+handle-group-update`'s handling of the `%remove` case? Right now the `+listen-to-*` and `+leave-from-*` arms deal exclusively with lower-level subscription management, all the "when to auto-join/leave" semantics happen in the update/action arms.